src/oscar/apps/catalogue/views.py

```diff
@@ -28,8 +28,10 @@ class ProductDetailView(DetailView):
     # Whether to redirect to the URL with the right path
     enforce_paths = True
 
-    # Whether to redirect child products to their parent's URL
-    enforce_parent = True
```

Comment:
This will break compatibility - I think it would be better not to change it unless there is a very good justification for the original default no longer being appropriate?

If we do make the change then the [documentation](http://django-oscar.readthedocs.io/en/releases-1.5/topics/modelling_your_catalogue.html#parent-and-child-products) needs updating.

Reply:
> This will break compatibility - I think it would be better not to change it unless there is a very good justification for the original default no longer being appropriate?

If we don't change value of the property, change on the product detail page (list of variants with absolute URLs to their pages), which replaced variant selection from the dropdown, will not work. I also don't find redirection on variant selection in the dropdown best solution, although it could allow to preserve backwards compatibility.

Reply:
Or we can introduce it within other breaking changes in 2.0. WDYT?

Reply:
Again, we can properly display variant-specific field values only on separate page, thus default property update looks inevitable. Those who decide to update data within the variant selection, could override this property value in the project.